app/services/twilio/twilio_text_messenger.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Twilio
+  class TwilioTextMessenger < ApplicationService
+    attr_reader :message
+
+    def initialize(message)
+      @message = message
+    end
+
+    def call
+      client = Twilio::REST::Client.new
```

Comment:
move `Twilio::REST::Client.new` to separate private method

Reply:
done